### DISCUSSION TITLE
How To Add New Flavors

### DISCUSSION THREAD

**Discussion by jag3773** [bible-technology/scripture-burrito#269] [Next]
How would @jag3773 add "Translation Notes" as a para-scriptural FlavorType?

First, how to use a custom flavor "on my own" ? Do I get schema validation of my Flavor in any way?

Second, how to propose this custom flavor for inclusion in the SB standard? What happens if you drop "x-" when adding to the schema, does it become an alias?

The goal would be to have a simple workflow defined for both of the above. Secondarily, if needed, make any requisite schema changes in #261.

**Comment by jtauber** [comment]
We currently have https://docs.burrito.bible/en/v0.3.0/flavors/extending.html

It has XML-specific language in it but worth reviewing to see what else needs to changed / added to.

**Comment by jtauber** [comment]
I'm just finishing up documentation on this now based on what I did for the x-wordAlignment custom flavor
